dspace-api/src/main/java/org/dspace/core/I18nUtil.java

```diff
@@ -224,17 +224,17 @@ public static Locale getSupportedLocale(Locale locale)
 
 
     /**
-     * Get the appropriate localized version of input-forms.xml according to language settings
+     * Get the appropriate localized version of submission-forms.xml according to language settings
      *
      * @param locale
-     *        Locale, the local to get the input-forms.xml for
-     * @return String - localized filename for input-forms.xml
+     *        Locale, the local to get the submission-forms.xml for
+     * @return String - localized filename for submission-forms.xml
      */
     public static String getInputFormsFileName(Locale locale)
```

Comment:
Do we still need this method? Won't all i18n related stuff be done by the UI?

Reply:
This will be done in the second stage. Thank you!